src/includes/enriching-events/import/javascript.ember.mdx

```diff
@@ -0,0 +1,3 @@
+```javascript
+import { InitSentryForEmber } from "@sentry/ember";
```

Comment:
@k-fish I think this has to change, is there `import * as Sentry from "@sentry/ember";`

Reply:
If so, we should fix it both here and in the Config sample, since that's where I copied it from.

Reply:
Yep `import * as Sentry from "@sentry/ember";` should work, you'll have to change it in the config sample, or I can if you like.

Reply:
Actually I'll just make a commit and you can revert it / edit it or whatever suits.

Reply:
thanks, @k-fish